resources/dicts/patrols/other_clan_hostile.json

```diff
@@ -586,7 +586,7 @@
         ],
         "antag_fail_outcomes": [
             {
-                "text": "How dare they trespass! p_l bounds after the scent, hoping to find the source of it, eventually discovering a limping o_c_n warrior who hastily explains that a dog had chased them into c_n's territory. p_l is still angry, but they rein in their hostility and tensely return the cat to o_c_n territory.",
+                "text": "How dare they trespass! p_l bounds after the scent, hoping to find the source of it, eventually discovering a limping o_c_n warrior who hastily explains that a dog had chased them into c_n's territory. p_l is still angry, but {PRONOUN/p_l/subject} rein in their hostility and tensely return the cat to o_c_n territory.",
```

Comment:
tag "reins" and "returns"

Reply:
![image](https://github.com/ClanGenOfficial/clangen/assets/126827015/47a1a9b2-edb5-4510-a49a-bca5cf7c941e)
i dont believe rein and return are tagged here? i thought it meant 'they' as the whole patrol